### PR TITLE
Feature login

### PR DESCRIPTION
**Pull request by trandaison**
I have finish login feature. This include some HTML/CSS files and `layouts blade` files so it have 21 files changed. But actually only 14 files have been changed, the others have nothing to review (I think so).
Please review and merge this one. Thank you!

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <!-- Meta, title, CSS, favicons, etc. -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Login</title>
```
Comment (dzungnguyenh) [inline resources/views/auth/login.blade.php]: [rank B] why did you put "Login" here?

File: resources/views/auth/passwords/email.blade.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
+@extends('layouts.app')
+
+<!-- Main Content -->
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
```
Comment (dzungnguyenh) [inline resources/views/auth/passwords/email.blade.php]: [rank B] Please use defined text in language file

File: resources/views/auth/passwords/email.blade.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                        <div class="alert alert-success">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    <form class="form-horizontal" role="form" method="POST" action="{{ url('/password/email') }}">
+                        {{ csrf_field() }}
+
+                        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                            <label for="email" class="col-md-4 control-label">E-Mail Address</label>
```
Comment (dzungnguyenh) [inline resources/views/auth/passwords/email.blade.php]: [rank B] languages issue

File: resources/views/auth/passwords/email.blade.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="col-md-6 col-md-offset-4">
+                                <button type="submit" class="btn btn-primary">
+                                    <i class="fa fa-btn fa-envelope"></i> Send Password Reset Link
```
Comment (dzungnguyenh) [inline resources/views/auth/passwords/email.blade.php]: [rank B] languages issue

File: resources/views/auth/passwords/reset.blade.php [hunk 5]
```diff
@@ -0,0 +1,70 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
```
Comment (dzungnguyenh) [inline resources/views/auth/passwords/reset.blade.php]: [rank B] please check all labels content

**Comment by trandaison** [comment]
@at-dungnh 
I've fixed some language issues and remove some files unused. 
Please review and merge this branch. Thank you!

**Comment by dzungnguyenh** [comment]
@at-sontd LGTM. Thank you.

**Pull request merged**